hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java

```diff
@@ -714,13 +714,40 @@ private String getQueueClassPrefix() {
     return CommonConfigurationKeys.IPC_NAMESPACE + "." + port;
   }
 
+  @Deprecated
```

Comment:
Is there a problem with simply deleting it?

Reply:
The Server class is public API. It's used by several other projects, including Tez and Ratis. I haven't checked how it is used but it would be great to avoid breakage.

Reply:
@jojochuang Thanks for your explanation. I understand.